### PR TITLE
[ADD] auth_sms

### PR DESCRIPTION
**Pull request by hbrunn**
This module allows you to use SMS as second factor for user authentication.

While SMS is not the most secure way of delivering a secret, it's still safer
than no multi factor authentication at all.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
What about https://github.com/OCA/server-tools/tree/10.0/auth_totp ?

**Comment by hbrunn** [comment]
it exists ;-) would be my first choice too indeed, but it's not feasible for the customer. Non-smartphone users and the like

File: auth_sms/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Therp BV <https://therp.nl>
```
Comment (NL66278) [inline auth_sms/__init__.py]: No more copyright in __init__.py files.

**Review by NL66278 — CHANGES_REQUESTED** [review]
:+1: Looks mostly good to me, but please remove copyright from init files.

**Pull request closed without merging**